migration/identity-migration/src/main/java/io/camunda/migration/identity/config/saas/StaticEntities.java

```diff
@@ -34,10 +35,15 @@ public class StaticEntities {
           new CreateRoleRequest(OPERATIONS_ENGINEER_ROLE_ID, "Operations Engineer", ""),
           new CreateRoleRequest(TASK_USER_ROLE_ID, "Task User", ""),
           new CreateRoleRequest(VISITOR_ROLE_ID, "Visitor", ""));
-
   public static final List<CreateAuthorizationRequest> ROLE_PERMISSIONS =
       List.of(
           // DEVELOPER
+          new CreateAuthorizationRequest(
```

Comment:
I reordered the entries by resource type alphabetically to be aligned with the order we have here https://docs.google.com/document/d/1XqG9Y9SXvakwcJ7ZheE8lgd6X0PMWSuyH4IbYTzLYeE/edit?pli=1&tab=t.0#bookmark=id.xx922nas68kh